src/open_producten/utils/filters.py

```diff
@@ -66,3 +105,120 @@ def filter(self, qs, value):
 
         qs = self.get_method(qs)(**{lookup: value, language_lookup: language_code})
         return qs
+
+
+class Operators(models.TextChoices):
+    exact = "exact", _("gelijk aan")
+    gt = "gt", _("grooter dan")
+    gte = "gte", _("grooter dan of gelijk aan")
+    lt = "lt", _("kleiner dan")
+    lte = "lte", _("kleiner dan of gelijk aan")
+    icontains = "icontains", _("case-insensitive partial match")
+    in_list = "in", _("in een lijst van waarden gescheiden door `|`")
```

Comment:
For constants it is better to write with capital letters
```suggestion
    EXACT = "exact", _("gelijk aan")
    GT = "gt", _("grooter dan")
    GTE = "gte", _("grooter dan of gelijk aan")
    LT = "lt", _("kleiner dan")
    LTE = "lte", _("kleiner dan of gelijk aan")
    ICONTAINS = "icontains", _("case-insensitive partial match")
    IN_LIST = "in", _("in een lijst van waarden gescheiden door `|`")
```